### PR TITLE
Stop energy blocks from being drawn upon the YOU LOSE screen #40

### PR DESCRIPTION
**Pull request by Alecxannder-Sonnenberg**
Deals with Issue #40

### CODE REVIEW

**Comment by Alecxannder-Sonnenberg** [comment]
Sorry for procrastinating on the issue, seems like the bug is fixed now, I hope everything is in order.

File: src/ttsnake.c [hunk 1]
```diff
@@ -356,6 +357,29 @@ void showscene (scene_t* scene, int number, int menu)
...
+			  break;
+		  }
+	  }
+	  if(isValid == 1){
+		  i++;
+	  }
+
+  	}while(i < MAX_ENERGY_BLOCKS);
+
+=======
```
Comment (matheushjs) [inline src/ttsnake.c]: small error here, but I'll try to fix it myself

**Review by matheushjs — CHANGES_REQUESTED** [review]
I am unable to compile. Did you manage to do so?

There are variables being redefined, and two blocks of code that do the same thing.

I think you might have messed things up when resolving conflicts of a merge.

The error:

```
ttsnake.c: In function ‘more_snacks’:
ttsnake.c:382:7: error: redefinition of ‘i’
   int i = 0, j, isValid = 0;
       ^
ttsnake.c:361:7: note: previous definition of ‘i’ was here
   int i = 0, j = 0, isValid;
       ^
ttsnake.c:382:14: error: redeclaration of ‘j’ with no linkage
   int i = 0, j, isValid = 0;
              ^
ttsnake.c:361:14: note: previous definition of ‘j’ was here
   int i = 0, j = 0, isValid;
              ^
ttsnake.c:382:17: error: redeclaration of ‘isValid’ with no linkage
   int i = 0, j, isValid = 0;
                 ^~~~~~~
ttsnake.c:361:21: note: previous declaration of ‘isValid’ was here
   int i = 0, j = 0, isValid;
                     ^~~~~~~
```

**Comment by Alecxannder-Sonnenberg** [comment]
I most likely screwed things up when resolving the conflicts, will try to find it out.

File: src/ttsnake.c [hunk 2]
```diff
@@ -356,6 +357,29 @@ void showscene (scene_t* scene, int number, int menu)
...
  	int i = 0, j, isValid = 0;
```
Comment (Alecxannder-Sonnenberg) [inline src/ttsnake.c:383]: This seems to be where I messed up

File: src/ttsnake.c (hunk 1 above)
Comment (Alecxannder-Sonnenberg) [reply src/ttsnake.c]: > I am unable to compile. Did you manage to do so?
> 
> There are variables being redefined, and two blocks of code that do the same thing.
> 
> I think you might have messed things up when resolving conflicts of a merge.
> 
> The error:
> 
> ```
> ttsnake.c: In function ‘more_snacks’:
> ttsnake.c:382:7: error: redefinition of ‘i’
>    int i = 0, j, isValid = 0;
>        ^
> ttsnake.c:361:7: note: previous definition of ‘i’ was here
>    int i = 0, j = 0, isValid;
>        ^
> ttsnake.c:382:14: error: redeclaration of ‘j’ with no linkage
>    int i = 0, j, isValid = 0;
>               ^
> ttsnake.c:361:14: note: previous definition of ‘j’ was here
>    int i = 0, j = 0, isValid;
>               ^
> ttsnake.c:382:17: error: redeclaration of ‘isValid’ with no linkage
>    int i = 0, j, isValid = 0;
>                  ^~~~~~~
> ttsnake.c:361:21: note: previous declaration of ‘isValid’ was here
>    int i = 0, j = 0, isValid;
>                      ^~~~~~~
> ```

Turns out when I tried to resolve conflicts I didn't remove the previous version of more_snacks() so variables where declared twice, I removed the previous iteration and matched it to the develop branch, will try testing it now.

**Comment by Alecxannder-Sonnenberg** [comment]
Tested after update, compiled fine and the bug was resolved, probably no more screw-ups in the code.

**Comment by matheushjs** [comment]
Tested and is working

**Pull request merged**